lib/Common/DataStructures/FixedBitVector.h

```diff
@@ -317,7 +318,7 @@ void BVFixed::SetRange(Container* value, BVIndex start, BVIndex len)
     {
         // Calculation above might overflow the original container.
         // normalize the overflow value. LE only
-        temp = (*((char*)bits)) + (*(((char*)bits) + 1));
+        //temp = (*((char*)bits)) + (*(((char*)bits) + 1));
```

Comment:
Review: This variable looks like it's never used. Are we setting a flag bit for the calling function?